core/src/test/java/kafka/log/remote/RemoteLogManagerTest.java

```diff
@@ -1018,8 +1018,8 @@ void testRemoteLogManagerRemoteMetrics() throws Exception {
                         safeLongYammerMetricValue("RemoteCopyLagBytes,topic=" + leaderTopic),
                         safeLongYammerMetricValue("RemoteCopyLagBytes")));
         TestUtils.waitForCondition(
-                () -> 1 == safeLongYammerMetricValue("RemoteCopyLagSegments") && 1 == safeLongYammerMetricValue("RemoteCopyLagSegments,topic=" + leaderTopic),
-                String.format("Expected to find 1 for RemoteCopyLagSegments metric value, but found %d for topic 'Leader' and %d for all topics.",
+                () -> 0 == safeLongYammerMetricValue("RemoteCopyLagSegments") && 0 == safeLongYammerMetricValue("RemoteCopyLagSegments,topic=" + leaderTopic),
```

Comment:
This is a bit weird to me, as the value of `RemoteCopyLagBytes` is not zero. Maybe we should change line#1003 to return `2` instead of `1`

Reply:
Updated, PTAL.